src/Foundatio/Caching/HybridCacheClient.cs

```diff
@@ -88,19 +88,17 @@ public class HybridCacheClient : ICacheClient {
             CacheValue<T> cacheValue;
             bool requiresSerialization = TypeRequiresSerialization(typeof(T));
```

Comment:
We should be able to get rid of this as it's no longer being used? If this method isn't being used anymore can we remove it as well.